db/migrate/20161218183833_simpler_dates.rb

```diff
@@ -0,0 +1,25 @@
+class SimplerDates < ActiveRecord::Migration
+  def up
+    execute 'update podcasts set published_at = pub_date where pub_date is not null and pub_date > published_at'
+    execute 'update podcasts set updated_at = last_build_date where last_build_date is not null and last_build_date > updated_at'
+    remove_column :podcasts, :last_build_date
+    remove_column :podcasts, :pub_date
+
+    execute 'update episodes set published_at = published where published is not null and published > published_at'
+    execute 'update episodes set updated_at = updated where updated is not null and updated > updated_at'
+    remove_column :episodes, :released_at
```

Comment:
I see no way they are different concepts, but I am looking up published_at and only overwriting if published is greater then published_at